dashboard/client/src/pages/FunctionLogPage.js

```diff
@@ -28,7 +27,11 @@ export class FunctionLogPage extends Component {
       this.setState({ isLoading: false, log: res });
     });
   }
-
+  onEditorLoad(editor) {
+    let n = editor.getSession().getValue().split("\n").length - 2;
```

Comment:
Can we do this without getting the splitting the text value? I'd really like to avoid that since the logs could be large.

Reply:
As per the ace [editor docs](https://ace.c9.io/#nav=api&api=editor).

I think we can do something like the following:

```
 editor.scrollToLine(editor.getSession().getLength());
```

Could you confirm and update if possible @jaigouk ?

Reply:
@kenfdev it seems like that's also working

![2018-09-18 at 00 54](https://user-images.githubusercontent.com/6720/45655300-b3b11380-badf-11e8-8101-6208bb26a2af.gif)